### DISCUSSION TITLE
Is there a way to use variables within evaluate FQL?

### DISCUSSION THREAD

**Discussion by redheaddev** [postmanlabs/postman-flows#433] [Ideas]
I'd like to be able and filter incoming data for a specific value stored in a variable.
example FQL: data[id=parentId]

data = variable data from prior step
id = attribute on data object
parentId = variable set at earlier step in flow

Note: If I substitute parentid with a string literal like "1234" the query returns data. 

In summary, is there a way to use the parentId token in FQL?

**Comment by viko42** [comment]
I'm also looking for that.
Currently, I bypass that problem with using custom tests in my requests

**Comment by minoseah629** [comment]
a similar note is to use a variable within the context of evaluate block.

<img width="741" alt="image" src="https://user-images.githubusercontent.com/1980068/214123893-682b0956-7c24-4c69-bdc3-7367a07c083f.png">
